### PR TITLE
Add script to check OnSeekMediaClockTimer with minutes value lower bound

### PR DESCRIPTION
**Pull request by OVikhrov**
@aarotar Please review

### CODE REVIEW

File: test_scripts/API/OnSeekMediaClockTimer/ATF_P_OnSeekMediaClockTimer_seekTime_minutes_Lower_Bound.lua [hunk 1]
```diff
@@ -0,0 +1,94 @@
+--------------------------------------------------------------------------------------------
+-- Requirement summary:
```
Comment (aarotar) [inline test_scripts/API/OnSeekMediaClockTimer/ATF_P_OnSeekMediaClockTimer_seekTime_minutes_Lower_Bound.lua]: @OVikhrov, please update Requirement summary with related functional requirement also

File: test_scripts/API/OnSeekMediaClockTimer/ATF_P_OnSeekMediaClockTimer_seekTime_minutes_Lower_Bound.lua [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+-- [HMI_API] New OnSeekMediaClockTimer notification
+--
+-- Description:
+-- [HMI_API] <param name="minutes" type="Integer" minvalue="0" maxvalue="59" mandatory="true">
+-- [MOBILE_API] <param name="minutes" type="Integer" minvalue="0" maxvalue="59">
+--
+-- 1. Used preconditions:
+-- a) First SDL life cycle
+-- b) OnSeekMediaClockTimer notification allowed in preloaded file for default app
+-- c) App successfylly registered, consented and activated
```
Comment (aarotar) [inline test_scripts/API/OnSeekMediaClockTimer/ATF_P_OnSeekMediaClockTimer_seekTime_minutes_Lower_Bound.lua]: @OVikhrov, please check syntax in script

File: test_scripts/API/OnSeekMediaClockTimer/ATF_P_OnSeekMediaClockTimer_seekTime_minutes_Lower_Bound.lua [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+function Test:Preconditions_ActivateApp()
+commonSteps:ActivateAppInSpecificLevel(self, self.applications["Test Application"])
+EXPECT_NOTIFICATION("OnHMIStatus", {hmiLevel = "FULL", systemContext = "MAIN"})
+end
+
+--[[ Test ]]
+commonFunctions:newTestCasesGroup("Test")
+function Test:TestStep_OnSeekMediaClockTimer_minutes_Value_Of_seekTime_Is_Lower_Bound()
+ self.hmiConnection:SendNotification("UI.OnSeekMediaClockTimer",{seekTime =  {hours = 0, minutes = 0, seconds = 33}, appID = self.applications["Test Application"]})
+
```
Comment (aarotar) [inline test_scripts/API/OnSeekMediaClockTimer/ATF_P_OnSeekMediaClockTimer_seekTime_minutes_Lower_Bound.lua]: @OVikhrov, plase avoid redundant empty lines

File: test_scripts/API/OnSeekMediaClockTimer/ATF_P_OnSeekMediaClockTimer_seekTime_minutes_Lower_Bound.lua (hunk 1 above)
Comment (OVikhrov) [inline test_scripts/API/OnSeekMediaClockTimer/ATF_P_OnSeekMediaClockTimer_seekTime_minutes_Lower_Bound.lua]: Accepted

File: test_scripts/API/OnSeekMediaClockTimer/ATF_P_OnSeekMediaClockTimer_seekTime_minutes_Lower_Bound.lua (hunk 2 above)
Comment (OVikhrov) [inline test_scripts/API/OnSeekMediaClockTimer/ATF_P_OnSeekMediaClockTimer_seekTime_minutes_Lower_Bound.lua]: Accepted

File: test_scripts/API/OnSeekMediaClockTimer/ATF_P_OnSeekMediaClockTimer_seekTime_minutes_Lower_Bound.lua (hunk 3 above)
Comment (OVikhrov) [inline test_scripts/API/OnSeekMediaClockTimer/ATF_P_OnSeekMediaClockTimer_seekTime_minutes_Lower_Bound.lua]: Accepted

**Comment by OVikhrov** [comment]
@aarotar Plese review iteration#2

**Pull request merged**